CMakeLists.txt

```diff
@@ -47,7 +47,7 @@ include (${PROJECT_SOURCE_DIR}/build/FindQt5Transitional.cmake)
 
 #include modules
 set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/build" ${CMAKE_MODULE_PATH})
-include (UsePkgConfig1)
+include (UsePkgConfig1) #defines MACRO(PKGCONFIG1 _package _minVersion _include_DIR _link_DIR _link_FLAGS _cflags)
```

Comment:
note that to prevent confusion about the differences in the parameter list for regular "PKGCONFIG", I have renamed the macro defined by UsePkgConfig1 as "PKGCONFIG1".  (PKGCONFIG1 uses _minVersion as 2nd argument, while PKGCONFIG doesn't).